### PR TITLE
fix: [PL-25636]: Re-positioning Use template btn in createPipeline modal

### PR DESCRIPTION
**Pull request by sunnykesh-harness**
##### Summary:

Issues as earlier if **Use Template button** is before **Git form fields** 

- By default clicking it will create Inline pipeline only.
- If user click it just after selecting Remote, all required remote Form fields are highlighted.

Fix:
Repositioning it at end of form as earlier intention. 

**Before fix:** 
Screenshot 2022-06-02 at 1 27 47 PM

**After fix:** 
Screenshot 2022-06-02 at 1 29 09 PM
Screenshot 2022-06-02 at 1 28 52 PM

Without Git Simplification enabled 
Screenshot 2022-06-02 at 1 29 54 PM

##### Jira Links:
https://harness.atlassian.net/browse/PL-25636

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/70-pipeline/components/PipelineStudio/CreateModal/PipelineCreate.tsx [hunk 1]
```diff
@@ -295,6 +278,29 @@ export default function CreatePipelines({
...
             ) : null}
 
+            {isGitSimplificationEnabled ? (
+              <Divider className={cx({ [css.gitSimplificationDivider]: storeType?.type === StoreType.INLINE })} />
+            ) : null}
+
+            {!isEdit && isPipelineTemplateEnabled && (
+              <Container padding={{ top: 'large' }}>
+                <Button
+                  text={'Start with Template'}
```
Comment (avinashmadhwani02) [inline src/modules/70-pipeline/components/PipelineStudio/CreateModal/PipelineCreate.tsx:288]: Can we move this to string framework?

Comment (sunnykesh-harness) [reply src/modules/70-pipeline/components/PipelineStudio/CreateModal/PipelineCreate.tsx:288]: yes, let me change it.

**Review by chetanvekariya-harness — DISMISSED** [review]

**Comment by sunnykesh-harness** [comment]
trigger cypress

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Comment by sunnykesh-harness** [comment]
retrigger jest

**Review by avinashmadhwani02 — APPROVED** [review]

**Comment by sunnykesh-harness** [comment]
trigger cypress

**Pull request merged**